src/main/resources/assets/neoforge/lang/en_us.json

```diff
@@ -274,7 +274,7 @@
   "neoforge.network.feature_flags.entry_mismatch": "The server and client have different sets of custom FeatureFlags. Make sure you are using the same mod and NeoForge versions as the server. See the log for more details",
   "neoforge.network.feature_flags.no_vanilla_server": "This client does not support vanilla servers as it has custom FeatureFlags",
 
-  "neoforge.attribute.debug.base": "[Entity: %s | Item: %s]",
+  "neoforge.attribute.debug.base": "[%s%s]",
```

Comment:
Would a seperator in between these two values not make sense?

So something like: `[%s | %s]`

Reply:
Ultimately my conclusion here was that the symbolism doesn't matter much. Someone that's reading the values here to glean useful information knows in advance that there are two values (the entity base and the item base), and no amount of hints/symbolics will help people that do not know what the values mean. So I went with the least verbose / most compact option.